src/app/loadout-drawer/LoadoutDrawerContents.tsx

```diff
@@ -332,6 +332,8 @@ export function fillLoadoutFromEquipped(
   itemsByBucket: { [bucketId: string]: DimItem[] },
   dimStore: DimStore,
   onUpdateLoadout: (loadout: Loadout) => void,
+  // This is a bit dangerous as it is only used from the new loadout edit drawer and
+  // has special handling that would break the old loadout drawer
```

Comment:
Right, that's why it's optional :-D

Reply:
Yeah just figured I would call it out incase any one else that was unfamiliar came across it. I had find all the references and see where it was getting called from.